about.html

```diff
@@ -12,62 +12,96 @@
+        <div class='container-1'>    
+            <img class="item" src="img/lambda-black.png" alt="img Lambda">
+            <a class='item' href="#">Home</a>
+            <a class='item' href="about.html">About</a>
+            <a class='item' href="#">Products</a>
+            <a class='item' href="#">Blog</a>
+            <a class='item' href="#">Contact</a>    
+        </div>
+
+        <img class="imgJumbo" src="img/jumbo-about.png" alt="img Jumbo">
+  
+  
+  
+<br>  
+  <div class="container-about-page">
+     
+      <h2>About</h2>
 
+        <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+  </div>
+        <br>
 
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
+<hr>        
+    <div class='text-boxes'>
+        
+      <div class='text-box'>
+        <img src="img/about-plan.png" alt="strategy">
+        
+          <h2>Strategy</h2>
+        
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            <button>Learn More</button>
+      </div>
+<hr>
+      <div class='text-box'>
+        <img src="img/about-working.png" alt="strategy">
+        
+          <h2>How We Work</h2>
+        
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Learn More
+            <button>Learn More</button>
+    <hr>
+  </div>
 
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+  <div class='text-box'>
+        
+          <img src="img/about-office.png" alt="strategy">
+          
+            <h2>Places We Work</h2>
 
-    Learn More
+              <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
 
-    Let's Work Together
+              <button>Learn More</button>
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+        </div>      
+<br>
+        <hr>          
+        <div class='text-box'>    
         
+        <img src="img/about-meeting.png" alt="strategy">
```

Comment:
So I noticed all the `<img>` alts are the same. The alt is used by screen readers and Search Engine Optimization machines as an alternative when the image doesn't load, as well as to be read to users with accessibility needs (i.e visually impaired). These should definitely be more specific and apply to the image they are on.